controllers/activemqartemis_controller_test.go

```diff
@@ -9322,6 +9322,66 @@ var _ = Describe("artemis controller", func() {
 			Expect(k8sClient.Delete(ctx, &crd)).To(Succeed())
 		})
 
+		It("-bp suffix secret broker-n support", Label("broker-n-bp-secret"), func() {
+
+			ctx := context.Background()
+
+			_, bpSecret1 := DeploySecret(defaultNamespace, func(candidate *corev1.Secret) {
+				candidate.Name = "config-1-bp"
+				candidate.StringData = map[string]string{
+					"journal1.properties":       `journalFileSize=12345`,
+					"broker-0.first.properties": `globalMaxSize=512m`,
+				}
+			})
+
+			_, bpSecret2 := DeploySecret(defaultNamespace, func(candidate *corev1.Secret) {
+				candidate.Name = "config-2-bp"
+				candidate.StringData = map[string]string{
+					"journal2.properties":        `journalMinFiles=3`,
+					"broker-0.second.properties": `journalFileSize=12M`,
+				}
+			})
+
+			By("Deploying the CRD")
+			_, crd := DeployCustomBroker(defaultNamespace, func(candidate *brokerv1beta1.ActiveMQArtemis) {
+				candidate.Spec.DeploymentPlan.Size = common.Int32ToPtr(1)
```

Comment:
I think this test needs to go further. deploy two instanced of the ss. and have in config-1-bp
					"broker-0.first.properties": `globalMaxSize=512m`,

and config-2-bp:
					"broker-1.second.properties": `globalMaxSize=12m`,

Then verify via jolokia that, ordinal 0 has globalMax 512m and ordinal 1 has 12M 

just validating the Status presence of the property names may not be sufficient

Reply:
whether there is a need for first and second .properties, or whether the could both use broker-N.globalMem.properties: globalMaxSixe=X 
it may also help for the doc. Same comment in the doc section.

Reply:
+1